device-server/src/main/kotlin/com/badoo/automation/deviceserver/ios/simulator/Simulator.kt

```diff
@@ -91,20 +91,20 @@ class Simulator (
             val elapsed = measureTimeMillis {
                 prepare(clean = true)
             }
-            logger.info(logMarker, "Device ${Simulator@this.toString()} ready in ${elapsed / 1000} seconds")
+            logger.info(logMarker, "Device ${this@Simulator} ready in ${elapsed / 1000} seconds")
         }
     }
 
     private fun prepare(timeout: Duration = PREPARE_TIMEOUT, clean: Boolean) {
-        logger.info(logMarker, "Starting to prepare ${Simulator@this}. Will wait for ${timeout.seconds} seconds")
+        logger.info(logMarker, "Starting to prepare ${this@Simulator}. Will wait for ${timeout.seconds} seconds")
```

Comment:
Maybe we should do this in some other pull request (unrelated changes)?

Reply:
agreed